packages/datagateway-download/src/downloadCart/downloadCartTable.component.tsx

```diff
@@ -44,26 +56,36 @@ interface DownloadCartTableProps {
 const DownloadCartTable: React.FC<DownloadCartTableProps> = (
   props: DownloadCartTableProps
 ) => {
-  const settings = React.useContext(DownloadSettingsContext);
+  const { fileCountMax, totalSizeMax, apiUrl, facilityName } = React.useContext(
+    DownloadSettingsContext
+  );
 
   const [sort, setSort] = React.useState<{ [column: string]: Order }>({});
   const [filters, setFilters] = React.useState<{
     [column: string]: { value?: string | number; type: string };
   }>({});
 
-  const fileCountMax = settings.fileCountMax;
-  const totalSizeMax = settings.totalSizeMax;
-
   const [showConfirmation, setShowConfirmation] = React.useState(false);
 
   const { data: isTwoLevel } = useIsTwoLevel();
   const { mutate: removeDownloadCartItem } = useRemoveEntityFromCart();
   const { mutate: removeAllDownloadCartItems, isLoading: removingAll } =
     useRemoveAllFromCart();
-  const { data, isFetching: dataLoading } = useCart();
+  const { data: cartItems, isFetching: isFetchingCart } = useCart();
+  const { data: facilityCycles, isFetching: isFetchingFacilityCycles } =
```

Comment:
I'm pretty sure this is no longer needed right? Since we get facility cycles from the investigation link now